src/main/kotlin/no/nav/persondataapi/service/BrukertilgangService.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.persondataapi.service
+
+import no.nav.security.token.support.core.context.TokenValidationContext
+import no.nav.security.token.support.core.context.TokenValidationContextHolder
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+
+@Service
+class BrukertilgangService(
+    val tokenValidationContextHolder: TokenValidationContextHolder,
+    val tilgangService: TilgangService,
+) {
+    fun harBrukerTilgangTilIdent(ident: String): Boolean {
```

Comment:
Dette er en ny service for å validere at tokenet til brukeren har tilgang til å se informasjon om et fødselsnummer.